src/nft/utils/address.ts

```diff
@@ -1,7 +1,7 @@
 import { isAddress } from '@ethersproject/address'
 
 /**
- * Shortens an Ethereum address by N characters
+ * Shortens an Ethereum address by N characters (search keywords: .slice .truncate)
```

Comment:
can this be moved to a general util file? maybe a new file called `src/utils/addresses.ts` since this isnt unique to just the NFT flow

Reply:
we actually also already have this one 🙈 https://github.com/Uniswap/interface/blob/f8dd0f7349c21319baf56fa78605fe8de5bc23ad/src/utils/index.ts

should I look into unifying?

Reply:
oh seeing thats a "0x" prefix only

Reply:
nope they are the same basically but one errors, i'll check if i can swap it out safely